minidump/src/minidump.rs

```diff
@@ -4636,7 +4634,9 @@ fn utf16_to_string(data: &[u16]) -> Option<String> {
     let len = data.iter().take_while(|c| **c != 0).count();
     let s16 = &data[..len];
     let bytes = unsafe { slice::from_raw_parts(s16.as_ptr() as *const u8, s16.len() * 2) };
-    UTF_16LE.decode(bytes, DecoderTrap::Strict).ok()
+    encoding_rs::UTF_16LE
```

Comment:
Is this code correct? It's assuming LE.

Reply:
I think this place and minidump-synth both assumed LE, I thought it was better to just keep the code mostly as it was before the change, but I just assumed it was because....no one really cares about BE any longer since there's no one running BE in a meaningful capacity.